test/e2e/tests/confirmations/signatures/sign-typed-data-v4.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import { withRedesignConfirmationFixtures } from '../helper-fixture';
+import {
+  DAPP_HOST_ADDRESS,
+  WINDOW_TITLES,
+  openDapp,
+  unlockWallet,
+} from '../../../helpers';
+import { Ganache } from '../../../seeder/ganache';
+import { Driver } from '../../../webdriver/driver';
+
+describe('Confirmation Signature - Sign Typed Data V4', function (this: Suite) {
+  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) { return; }
```

Comment:
Minor, but even this env check could be behind a helper function such as `isRedesignEnabled` so we don't have to spread the knowledge of the ENV.

Reply:
hm.. interesting. this would be a new pattern. I'm content with checking the ENV directly. If we were to do this, and with consideration of the `redesignedConfirmationsEnabled` selector naming, we could add the check to `shared/modules/environment.ts` as

`const isEnvEnableConfirmationRedesign = process.env.ENABLE_CONFIRMATION_REDESIGN`

would this proposed update be preferred? @matthewwalsh0

Reply:
actually, with the following PR that'll be merged soon, we'll remove the ENV checks in this PR 
https://github.com/MetaMask/metamask-extension/pull/24466

Reply:
Yes we will remove the env (probably before this PR gets merged 😅)